notebooks/Misconfiguration_detection_framework_for_data_type_errors.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<p><code>get_typos()</code>  Looks very similar to the work in the other notebook "Misconfiguration_detection_framework_for_spelling_errors.ipynb". Is datatype misconfiguration dependent on the spelling error identification work? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/configuration-files-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
It's not dependent but just preprocessing steps to fix all the typos before working on data type errors. 
If it's confusing I can remove it.